src/test/ui/issues/issue-66923.rs

```diff
@@ -0,0 +1,13 @@
+fn main() {
```

Comment:
Can you please extend the filename with a short hint at what the test is about as well as a comment in the file itself explaining what this is supposed to test?

r=me rollup with that done.